operator/src/lib.rs

```diff
@@ -273,12 +273,10 @@ impl KafkaState {
                         node_labels.insert(String::from(APP_INSTANCE_LABEL), self.context.name());
                         // unwrap is ok to use here, as this comes from a hard coded object and should
                         // really not fail!
+                        let version: &KafkaVersion = &self.kafka_cluster.spec.version;
```

Comment:
I'm only on my phone: is the comment above still valid?

Reply:
No, it is not.

Reply:
removed